rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -579,11 +551,19 @@ async fn run_controller(
                     Req::GetAdvancedSettings(tx) => {
                         tx.send(controller.advanced_settings.clone()).ok();
                     }
-                    Req::Quit => break,
                     Req::SchemeRequest(url) => if let Err(e) = controller.handle_deep_link(&url).await {
                         tracing::error!("couldn't handle deep link: {e:#?}");
                     }
-                    Req::SignIn => {
+                    Req::SystemTrayMenu(TrayMenuEvent::About) => controller.toggle_window("about")?,
```

Comment:
Is there no type-safe way of handling this? Should we at least extract `about` into a constant?

Reply:
That's a good idea, I'll try it